tests/format/flow/return-arrow/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -136,7 +136,7 @@ function f5(): string => string {}
 
 =====================================output=====================================
 const f1 = (): ((string) => string) => {};
-const f2 = (): (?(y) => { a: (b) => c }) => (0: any);
+const f2 = (): (?((y) => { a: (b) => c })) => (0: any);
```

Comment:
@vjeux Is this expected?

Reply:
I’m not sure. I haven’t dug into whether this was correct with this change or not.

Reply:
I don't think extra parens will break anything, let's merge as it is.

If we  found it's moot, we can improve later. WDYT?